packages/api/src/__tests__/useJurisdictionStatus.spec.tsx

```diff
@@ -0,0 +1,226 @@
+import * as React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import useAuthentication from '../hooks/useAuthentication';
+import usePOI from '../hooks/usePOI';
+import useJurisdictionStatus from '../hooks/useJurisdictionStatus';
+import APIProvider from '../APIProvider';
+
```

Comment:
These are the test cases which are checked:
![Uploading Screenshot 2023-11-13 at 2.47.28 PM.png…]()